test/trace/integrations/rails_integration_test.rb

```diff
@@ -0,0 +1,132 @@
+# Copyright 2017 OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "fileutils"
+require "open3"
+require "timeout"
+require "test_helper"
+require "opencensus/trace/integrations/rails"
+require "faraday"
+
+describe "Rails integration" do
+  module RailsTestHelper
+    APP_NAME = "railstest"
+    BASE_DIR = File.absolute_path File.dirname File.dirname File.dirname __dir__
+    TMP_DIR = File.join BASE_DIR, "tmp"
+    APP_DIR = File.join TMP_DIR, APP_NAME
+    RAILS_OPTIONS = %w(
+        --skip-yarn
+        --skip-action-mailer
+        --skip-active-record
+        --skip-action-cable
+        --skip-puma
+        --skip-sprockets
+        --skip-spring
+        --skip-listen
+        --skip-coffee
+        --skip-javascript
+        --skip-turbolinks
+        --skip-test
+        --skip-system-test
+        --skip-bundle
+      ).join ' '
+
+    class << self
+      def create_rails_app
+        puts "**** Creating test Rails app..."
```

Comment:
These steps could potentially be put into a [rails application template](http://guides.rubyonrails.org/rails_application_templates.html). Both of the `insert_after` lines could be accomplished in the template config.

This is not a blocker.

Reply:
That's a lot cleaner, thanks. Changed.